recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,92 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class QuiToque(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "quitoque.fr"
+
+    @staticmethod
+    def _get_text(element):
+        if element:
+            return normalize_string(element.get_text())
+        else:
+            return None
+
+    def _get_time(self, time_name):
+        times = self.soup.select("div,.recipe-infos-short .item-info")
+        total_time = None
+        for time in times:
+            if time_name in time.get_text():
+                total_time = self._get_text(time).replace(time_name, "")
+        return get_minutes(total_time)
+
+    def _get_nutrients(self, nutrient_name):
+        nutrient_element = self._nutrient_list.find("p", string=nutrient_name).parent
+        return self._get_text(nutrient_element.find("p", class_="regular"))
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"}).get("content")
+
+    def author(self):
+        return "QuiToque"
+
+    def title(self):
+        return self._get_text(self.soup.find("h1", class_="title-2"))
+
+    def keywords(self):
+        product_tags = self.soup.find(id="product-tags").find_all(class_="badge")
+        keywords = {self._get_text(tag) for tag in product_tags}
+        return list(sorted(keywords))
```

Comment:
When possible, it's good to retain the original ordering of keywords/tags from the source webpage.  In some circumstances it can be a deliberate choice.

The logic to do that -- while also removing duplicates -- isn't entirely trivial, but fortunately we have a [utility function](https://github.com/hhursev/recipe-scrapers/blob/bb50062fc333042e50293f7ff5dd2472190015b5/recipe_scrapers/_utils.py#L258-L267) to help with this.

You should be able to modify this to something like:

```suggestion
    def keywords(self):
        product_tags = self.soup.find(id="product-tags").find_all(class_="badge")
        keywords = ",".join(self._get_text(tag) for tag in product_tags)
        return csv_to_tags(keywords)
```